tests/conftest.py

```diff
@@ -785,229 +787,203 @@ def fixture_stats_api(project_dir: Path) -> StatsAPI:
+    """
+    return "211101_A00187_0615_AHLG5GDRZZ"
+
+
+@pytest.fixture(name="tmp_flow_cell_name_no_sample_sheet")
+def fixture_tmp_flow_cell_name_no_sample_sheet() -> str:
+    """This is the name of a flow cell directory with the run parameters missing."""
+    return "170407_A00689_0209_BHHKVCALXX"
+
+
+@pytest.fixture(name="tmp_flow_cell_name_ready_for_demultiplexing_bcl2fastq")
+def fixture_tmp_flow_cell_name_ready_for_demultiplexing_bcl2fastq() -> str:
+    """This is the name of a flow cell directory with the run parameters missing."""
+    return "211101_A00187_0615_AHLG5GDRXY"
+
+
+@pytest.fixture(name="tmp_flow_cells_directory_no_run_parameters")
+def fixture_tmp_flow_cells_directory_no_run_parameters(
+    tmp_flow_cell_name_no_run_parameters: str, tmp_flow_cells_directory: Path
 ) -> Path:
     """This is a path to a flow cell directory with the run parameters missing."""
-    working_dir: Path = Path(flow_cell_runs_working_directory, bcl2fastq_flow_cell_dir.name)
-    working_dir.mkdir(parents=True)
-    return working_dir
+    return Path(tmp_flow_cells_directory, tmp_flow_cell_name_no_run_parameters)
 
 
-@pytest.fixture(name="demultiplex_ready_flow_cell")
-def fixture_demultiplex_ready_flow_cell(
-    flow_cell_working_directory: Path, bcl2fastq_flow_cell_dir: Path
+@pytest.fixture(name="tmp_flow_cells_directory_no_sample_sheet")
+def fixture_tmp_flow_cells_directory_no_sample_sheet(
+    tmp_flow_cell_name_no_sample_sheet: str, tmp_flow_cells_directory: Path
 ) -> Path:
-    """Return the path to a working directory that is ready for demultiplexing.
+    """This is a path to a flow cell directory with the sample sheet and run parameters missing."""
+    return Path(tmp_flow_cells_directory, tmp_flow_cell_name_no_sample_sheet)
 
-    This is a path to a flow cell directory with all the files necessary to start demultiplexing present.
-    """
-    existing_flow_cell: FlowCellDirectoryData = FlowCellDirectoryData(
-        flow_cell_path=bcl2fastq_flow_cell_dir
-    )
-    working_flow_cell: FlowCellDirectoryData = FlowCellDirectoryData(
-        flow_cell_path=flow_cell_working_directory
-    )
-    shutil.copy(
-        existing_flow_cell.sample_sheet_path.as_posix(),
-        working_flow_cell.sample_sheet_path.as_posix(),
-    )
-    shutil.copy(
-        str(DemultiplexingAPI.get_stderr_logfile(existing_flow_cell)),
-        str(DemultiplexingAPI.get_stderr_logfile(working_flow_cell)),
-    )
-    working_flow_cell.copy_complete_path.touch()
-    working_flow_cell.rta_complete_path.touch()
-    return flow_cell_working_directory
+
+@pytest.fixture(name="tmp_flow_cells_directory_ready_for_demultiplexing_bcl_convert")
+def fixture_tmp_flow_cells_directory_ready_for_demultiplexing_bcl_convert(
+    tmp_flow_cell_name_ready_for_demultiplexing_bcl_convert: str, tmp_flow_cells_directory: Path
+) -> Path:
+    """This is a path to a flow cell directory with the run parameters missing."""
```

Comment:
This line seem repeated for many fixtures. Is it relevant for all?

Reply:
at some point I copied fixtures to alter them, is an artefact I updated accordingly